### ISSUE TITLE
403 on start on a mac

### ISSUE THREAD

**Issue by acinader** [mongodb-js/runner#142]
I am unable to start mongodb-runner.

I think that this is because mongodb-runner gets a 403 when trying to download mongo.

I've dug around in mongo-version-manager and the related modules to no avail.

I also tried with `npm install mongodb-runner@latest`

pretty sure the issue is that the path is incorrect:` osx/mongodb-macos-x86_64-4.0.3.tgz`

but I don't know how to resolve that.

```
$ uname -a
Darwin dev 18.2.0 Darwin Kernel Version 18.2.0: Tue Oct  2 22:38:23 PDT 2018; root:xnu-4903.221.1~12/RELEASE_X86_64 x86_64
$ mkdir erase-me
$ cd erase-me
$ npm install mongodb-runner
$ ./node_modules/.bin/mongodb-runner start
◠ Starting a MongoDB deployment to test against...{ HTTPError: Response code 403 (Forbidden)
    at EventEmitter.ee.on.res (/Users/arthur/Desktop/erase-me/node_modules/got/index.js:250:24)
    at emitOne (events.js:116:13)
    at EventEmitter.emit (events.js:211:7)
    at Immediate.setImmediate [as _onImmediate] (/Users/arthur/Desktop/erase-me/node_modules/got/index.js:99:8)
    at runCallback (timers.js:794:20)
    at tryOnImmediate (timers.js:752:5)
    at processImmediate [as _immediateCallback] (timers.js:729:5)
  name: 'HTTPError',
  host: 'fastdl.mongodb.org',
  hostname: 'fastdl.mongodb.org',
  method: 'GET',
  path: '/osx/mongodb-macos-x86_64-4.0.3.tgz',
  protocol: 'https:',
  url: 'https://fastdl.mongodb.org/osx/mongodb-macos-x86_64-4.0.3.tgz',
  statusCode: 403,
  statusMessage: 'Forbidden',
  headers:
   { 'content-type': 'application/xml',
     'transfer-encoding': 'chunked',
     connection: 'close',
     date: 'Tue, 30 Oct 2018 22:20:55 GMT',
     server: 'AmazonS3',
     'x-cache': 'Error from cloudfront',
     via: '1.1 87a01244684ed0c627925ef48431f67f.cloudfront.net (CloudFront)',
     'x-amz-cf-id': 'Xb2h3ajDFRuM0ii8ImjsjsZVjxSnjMrvSN4073oesc16UBBuemsV_A==' } }
{ HTTPError: Response code 403 (Forbidden)
    at EventEmitter.ee.on.res (/Users/arthur/Desktop/erase-me/node_modules/got/index.js:250:24)
    at emitOne (events.js:116:13)
    at EventEmitter.emit (events.js:211:7)
    at Immediate.setImmediate [as _onImmediate] (/Users/arthur/Desktop/erase-me/node_modules/got/index.js:99:8)
    at runCallback (timers.js:794:20)
    at tryOnImmediate (timers.js:752:5)
    at processImmediate [as _immediateCallback] (timers.js:729:5)
  name: 'HTTPError',
  host: 'fastdl.mongodb.org',
  hostname: 'fastdl.mongodb.org',
  method: 'GET',
  path: '/osx/mongodb-macos-x86_64-4.0.3.tgz',
  protocol: 'https:',
  url: 'https://fastdl.mongodb.org/osx/mongodb-macos-x86_64-4.0.3.tgz',
  statusCode: 403,
  statusMessage: 'Forbidden',
  headers:
   { 'content-type': 'application/xml',
     'transfer-encoding': 'chunked',
     connection: 'close',
     date: 'Tue, 30 Oct 2018 22:20:55 GMT',
     server: 'AmazonS3',
     'x-cache': 'Error from cloudfront',
     via: '1.1 87a01244684ed0c627925ef48431f67f.cloudfront.net (CloudFront)',
     'x-amz-cf-id': 'Xb2h3ajDFRuM0ii8ImjsjsZVjxSnjMrvSN4073oesc16UBBuemsV_A==' } }

**Comment by acinader** [comment]
see: https://github.com/mongodb-js/version-manager/issues/147

**Comment by imlucas** [comment]
`npm i mongodb-runner@latest` and you'll be good to go. included in `4.4.0`. this `osx` ➡️ `macOS` change wasn't well documented and didn't even think to check `4.0.x` :/ Sorry about that!
